FirebasePerformance/Sources/FPRURLFilter.m

```diff
@@ -17,7 +17,7 @@
 
 #import "FirebasePerformance/Sources/FPRConsoleLogger.h"
 
-#import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCCTUploader.h"
+#import "GoogleDataTransport/GDTCORLibrary/Internal/GoogleDataTransportInternal.h"
```

Comment:
This line is needed in the Perf podspec - https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCrashlytics.podspec#L29

Reply:
This is done.